index.js

```diff
@@ -57,6 +60,18 @@ slackEvents.on("message", event => {
     const currentStep = actions.tempIncidents[userId].step;
     switch (currentStep) {
       case 0:
+        if (isSubjectAdequate(event.text) === false) {
```

Comment:
@bmuthoga Since `isSubjectAdequate` is returning a boolean either way, you can avoid comparing with `false` and just do
```js
if(!isSubjectAdequate(event.text)) { //... }
```

Reply:
Makes sense, thanks